lib/ansible/modules/network/vyos/vyos_config.py

```diff
@@ -100,6 +102,15 @@
   vyos_config:
     src: vyos.cfg
     backup: yes
+
+- name: for idempotency, use full-form commands
+  nxos_config:
```

Comment:
looks like typo. it should be vyos?

Reply:
Yes, it should. Thanks for catching that.